ui/settingsdlg.ui

```diff
@@ -825,6 +825,16 @@ The qube must be running to disable seamless mode; this setting is not persisten
               </item>
              </layout>
             </item>
+            <item row="1" column="0" colspan="2">
+             <widget class="QCheckBox" name="dvm_template_checkbox">
+              <property name="toolTip">
+               <string/>
+              </property>
+              <property name="text">
+               <string>Allow starting Disposable VMs from this qube</string>
```

Comment:
This is yet another example why we need to settle on a solid definitions of VM vs Qube. I'll comment on #1015